### ISSUE TITLE
No longer able to generate a new prefix, if cloud saves are enabled and have saves to sync, since the cloud save rework on the new client update

### ISSUE THREAD

**Issue by gwemmie** [ValveSoftware/Proton#5657]
Version: literally every Proton version, it's wine in general

So, here's a very minor issue I've been experiencing with proton since day 1 of it existing: if there is already a folder in compatdata named after the appid, then proton will assume the prefix is already there and refuse to generate a new one--even if the one that's "already there" is incomplete--which means proton itself with crash on start.

But when would there ever be such an incomplete prefix? Well, let's say I'm trying to clear out my prefix in order to let a different version of proton generate a new one from scratch. Sometimes, Steam's cloud save feature will notice that the prefix is gone, and regenerate *ONLY* the part of it that includes the save files for that game. Not the entire prefix--just the saves and the dir path leading up to them, and nothing else. An incomplete prefix.

This used to only happen ocassionally if I booted Steam up *after* deleting that prefix folder. Still not a major issue.

But now that the Steam client has come out with a new update reworking the cloud save feature to be much more transparent and robust, one consequence is that, for me, it's running a cloud sync step *before* running proton every single time I launch the game. Even if I launch it with my internet disconnected, the cloud save step still runs and still creates an empty folder with the appid in compatdata.

The end result is I genuinely cannot figure out how to clear out and replace a game's prefix anymore, without disabling cloud sync for that game. In fact, if I delete a prefix that I didn't have backed up somewhere (not what happened this time, fortunately), then disabling cloud saves might even be the only way to get that game to run ever again, in the client's current state.

**Comment by gwemmie** [comment]
OK, disabling Cloud Sync for this game is not enough. It's still running the cloud sync step on launch even with it disabled for some reason.

I think I'm screwed. I literally cannot run Elden Ring anymore. This silly cloud save feature won't turn off and won't allow proton to run.

**Comment by gwemmie** [comment]
I just tried this in Offline Mode and the behavior hasn't even changed... it literally ran the cloud sync step and restored my saves in Offline Mode (and then immediately gave a warning saying it had failed to do that, a lie). This is terrifying. You can permanently break a proton game across your entire account just by storing saves in the cloud at all.

**Comment by gwemmie** [comment]
Anyway, my guess for the obvious fix would be to make proton finish making the prefix, instead of just give up the instant it sees that one directory at the top level already there.

**Comment by gwemmie** [comment]
Hold on, I'm sorry--this entire thing happened the way it did because I forgot to set the +x bit on a proton python script that I had downloaded directly from someone working on their own fork, for testing purposes. My mistake.

I know that in the past, proton did have the issue I described, where it would refuse to finish an incomplete prefix already there. But once I set the chmod +x bit, everything went fine, even though the cloud sync step was generating that folder before proton ran. So I guess that's total proof that that past issue is gone!
